Source/Views/AlertView.swift

```diff
@@ -72,6 +73,15 @@ class AlertView: AlertControllerView {
         }
     }
 
+    private func createBorders() {
+        if let borderColor = self.visualStyle.borderColor {
+            self.layer.borderColor = borderColor.cgColor
+        }
+        if let borderWidth = self.visualStyle.borderWidth {
+            self.layer.borderWidth = borderWidth
+        }
+    }
```

Comment:
Could you update this to be written as:

```swift
self.layer.borderColor = self.visualStyle.borderColor
self.layer.borderWidth = self.visualStyle.borderWidth ?? 0
```

(`borderColor` is optional so no need to unwrap that first)